pkg/testing/manager_mock.go

```diff
@@ -22,7 +24,7 @@ import (
 func NewMockManager() *MockManager {
 	mgr := &MockManager{
 		sc:        runtime.NewScheme(),
-		log:       testing.NullLogger{},
+		log:       logr.New(logging.NullLogSink{}),
```

Comment:
@shubham1172 Any particular reason for this change?

Reply:
`github.com/go-logr/logr` no longer has `testing.NullLogger{}`, because of which we had to implement our own version and use it here.